src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -607,10 +607,8 @@ export default class UniversalDiscoveryModule extends Component {
         const isAlreadySelected = selectedContent.find((item) => item.ContentInfo.Content._id === data.ContentInfo.Content._id);
         const isOverLimit = !!this.props.selectedItemsLimit && selectedContent.length >= this.props.selectedItemsLimit;
         const contentTypeInfo = contentTypesMap[data.ContentInfo.Content.ContentType._href];
-        const isAllowedContentType =
-            !this.props.cotfAllowedContentTypes.length || this.props.cotfAllowedContentTypes.includes(contentTypeInfo.identifier);
 
-        if (isAlreadySelected || isOverLimit || !isAllowedContentType) {
```

Comment:
Is this fix is about removing the feature of checking if content type can be selected? It doesn't seem like a proper fix.